docs/source/get_started/tutorial.rst

```diff
@@ -0,0 +1,52 @@
+Tutorials
```

Comment:
I seemed the previous tutorials is under construction. 
I updates the contents by putting on the notebook links.

Reply:
This expanded doctest tutorial is great! Hopefully it will provide people a nice inline code sample example for them to work with :)